packages/mdc-slider/foundation.js

```diff
@@ -180,6 +186,27 @@ class MDCSliderFoundation extends MDCFoundation {
     this.adapter_.setThumbAttribute(strings.ARIA_VALUEMIN, String(this.min_));
   }
 
+  /** @return {boolean} */
+  isDisabled() {
+    return this.disabled_;
```

Comment:
Can you provide some info about why you store the disabled state in a property? I'd think that returning whether the classlist includes the disabled class would be more accurate.

Reply:
Was used for a check in the handlestart however i can just call the isDisabled instead so i agree. Changed!